web/src/components/services-page/service-list-item.tsx

```diff
@@ -0,0 +1,65 @@
+import Img, { FluidObject } from 'gatsby-image';
+import React from 'react';
+import { Box, Button, Container, Flex, Grid, Heading, Label } from 'theme-ui';
+import { Link } from '../custom-link';
+
+export interface ServiceListItemProps {
+  name: string;
+  title: string;
+  description: string;
+  url: {
+    navigationLink: string;
+    linkText: string;
+  };
+  serviceImage?: FluidObject;
+  personPhoto?: FluidObject;
+  textAlign: 'left' | 'right';
+  dividedPhoto?: FluidObject;
+}
+
+// TODO: fix image showing when Sanity will be done
+export const ServiceListItem: React.FC<{ data: ServiceListItemProps }> = ({
+  data,
+}) => (
+  <Container id={data.name} sx={{ mt: '10%' }}>
```

Comment:
```suggestion
  <Container id={data.name} sx={{ mt: 4 }}>
```

or

```suggestion
  <Container id={data.name} sx={{ mt: '4rem' }}>
```

What ever makes the most sense. Using percentage is discouraged since we have a size matrix.